doc/start/hardware-recommendations.rst

```diff
@@ -169,29 +184,43 @@ performance of sequential reads and writes.
    SSD in a test configuration to gauge performance. 
 
 Relatively inexpensive SSDs may appeal to your sense of economy. Use caution.
-Acceptable IOPS are not enough when selecting an SSD for use with Ceph. 
+Acceptable IOPS are not the only factor to consider when selecting an SSD for
+use with Ceph. 
 
-SSDs have historically been cost prohibitive for object storage, though
-emerging QLC drives are closing the gap.  HDD OSDs may see a significant
+SSDs have historically been cost prohibitive for object storage, but 
+emerging QLC drives are closing the gap. HDD OSDs may see a significant
 performance improvement by offloading WAL+DB onto an SSD.
 
-One way Ceph accelerates CephFS file system performance is to segregate the
-storage of CephFS metadata from the storage of the CephFS file contents. Ceph
-provides a default ``metadata`` pool for CephFS metadata. You will never have to
-create a pool for CephFS metadata, but you can create a CRUSH map hierarchy for
-your CephFS metadata pool that points only to a host's SSD storage media. See
-:ref:`CRUSH Device Class<crush-map-device-class>` for details.
+Partition Alignment
+~~~~~~~~~~~~~~~~~~~
+
+When using SSDs with Ceph, make sure that your partitions are properly aligned.
+Improperly aligned partitions suffer slower data transfer speeds than do
+properly aligned partitions. For more information about proper partition
+alignment and example commands that show how to align partitions properly, see
+`Werner Fischer's blog post on partition alignment`_.
+
+CephFS Metadata Segregation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One way that Ceph accelerates CephFS file system performance is by segregating
+the storage of CephFS metadata from the storage of the CephFS file contents.
+Ceph provides a default ``metadata`` pool for CephFS metadata. You will never
+have to create a pool for CephFS metadata, but you can create a CRUSH map
+hierarchy for your CephFS metadata pool that points only to a host's SSD
+storage media.  See :ref:`CRUSH Device Class<crush-map-device-class>` for
+details.
 
 
 Controllers
 -----------
 
 Disk controllers (HBAs) can have a significant impact on write throughput.
-Carefully consider your selection to ensure that they do not create
-a performance bottleneck.  Notably RAID-mode (IR) HBAs may exhibit higher
-latency than simpler "JBOD" (IT) mode HBAs, and the RAID SoC, write cache,
-and battery backup can substantially increase hardware and maintenance
-costs.  Some RAID HBAs can be configured with an IT-mode "personality".
+Carefully consider your selection of HBAs to ensure that they do not create a
+performance bottleneck. Notably, RAID-mode (IR) HBAs may exhibit higher latency
+than simpler "JBOD" (IT) mode HBAs (and the RAID SoC, write cache, and battery
+backup can substantially increase hardware and maintenance costs).  Some RAID
```

Comment:
suggest removing the parens

Reply:
I'll break it into two sentences. I have an affinity for parentheses because I think that they can be used to structure thoughts in English in a way that is denied to us through mere use the grammatical rules that govern relative clauses, but this affinity might be traceable back to an adolescent preference for the writing of Stephen King and it might not be shared by other readers of this language.

Reply:
Lovecraft >> SK :p but I would mostly not advise writing HPL style.  Mostly.